modules/extensions/galasa-extensions-parent/dev.galasa.auth.couchdb/src/main/java/dev/galasa/auth/couchdb/internal/CouchdbAuthStore.java

```diff
@@ -353,11 +353,43 @@ private void validateCouchdbResponseJson(String expectedUserNumber , PutPostResp
         }
     }
 
+    @Override
+    public IUser getUser(String userNumber) throws AuthStoreException {
+
+        try{
```

Comment:
Nitpick - inconsistent formatting between this try-catch and the try-catch in deleteUser, would be good to use the same formatting